### DISCUSSION TITLE
Get data from General Settings

### DISCUSSION THREAD

**Discussion by VojtaSim** [swellstores/community#545] [Help] [answered]
Hello,

is it possible to obtain data from General settings via Backend API or GraphQL? (Business details - legal business name, address, etc.)

**Comment by sags95** [comment]
Yes with the Backend API you should be able to retrieve this through the `/:clients/:self` endpoint

**Reply by ericingram** [reply]
@sags95 they might be referring to /settings/general

**Reply by VojtaSim** [reply]
I mean this ones 
<img width="1286" alt="Screenshot 2024-06-13 at 08 10 15" src="https://github.com/swellstores/community/assets/4440970/984dca10-9a80-4bb1-8836-b3a541a151f9">

**Reply by sags95** [reply, marked as answer]
Those details can be retrieved through the `/:clients/:self` endpoint
